src/Model/Asn.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace GeoIp2\Model;
+
+/**
+ * This class provides the GeoIP2 ASN model.
```

Comment:
Thanks.  In this case I thought the `GeoIP2` was a reference to the class name, but I know see that the case doesn't match anyway.